login-workflow/src/pages/create-account/create-account.component.html

```diff
@@ -72,7 +72,7 @@
                 [activeStep]="registrationUtils.getCurrentPage()"
             >
                 <button pxb-back-button mat-stroked-button color="primary" style="width: 100px;" (click)="goBack()">
-                    {{ translate.GENERAL.BACK_BUTTON }}
+                    {{ translate.GENERAL.CANCEL_BUTTON }}
```

Comment:
Is this showing the 'Cancel' button on every page?

Reply:
Yep you're right; this is fixed now.